Doc updates to include new FIPS Synthetics PL image (#27778)
DataDog/documentation: content/en/synthetics/platform/private_locations/_index.md

Private locations are Docker containers that you can install anywhere inside your private network. You can access the [private location worker image][101] on Docker hub. It can run on a Linux-based OS or Windows OS if the [Docker engine][102] is available on your host and can run in Linux containers mode.**\***

{{< site-region region="gov" >}}

> Customers who require FIPS support should use the [FIPS compliant image][26] on Docker hub.

Reviewer: ```suggestion
If you require FIPS support, use the [FIPS compliant image][26] on Docker hub.

[26]: https://hub.docker.com/repository/docker/datadog/synthetics-private-location-worker-fips/general
```

[23]: /continuous_testing/cicd_integrations/configuration
[24]: /account_management/rbac/granular_access
[25]: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/specifying-sensitive-data-tutorial.html

> [26]: https://hub.docker.com/repository/docker/datadog/synthetics-private-location-worker-fips/general

Reviewer: ```suggestion
```
